jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
I'd prefer to use something pre-baked for caching. `functools.lrucache` with the backport module, say?